inference-engine/include/ie_extension.h

```diff
@@ -102,27 +99,7 @@ class INFERENCE_ENGINE_API_CLASS(Extension) : public IExtension {
     /**
      * @brief A SOPointer instance to the loaded templated object
      */
+    details::SharedObjectLoader::Ptr so;
     details::SOPointer<IExtension> actual;
```

Comment:
I suppose `details::SOPointer<IExtension> actual;` already contains link to `details::SharedObjectLoader::Ptr`, is not it?

Reply:
removed